definitions/procedures/backup/accessibility_confirmation.rb

```diff
@@ -0,0 +1,16 @@
+module Procedures::Backup
```

Comment:
If you move this outside of backup module into its own module (maybe something like UserInput or Confirmation) , I can use it as well as I have to ask the same exact question

Reply:
Actually the restore warning includes dropping the databases. Is there a way we can have a this accept a string as a question parameter so it can be used by restore (and other scenarios) with a custom question?

Reply:
This would bring possible complexity to how the procedure is built. We decided to keep this as is for now.